src/plugins/intel_cpu/src/config.cpp

```diff
@@ -287,6 +296,7 @@ void Config::updateProperties() {
     if (!_config.empty())
         return;
 
+    using namespace InferenceEngine;
```

Comment:
Why?! New plugin should use only new API!

Reply:
It will be handled in properties refactor PR: https://github.com/openvinotoolkit/openvino/pull/20022